docs/SimpleList.md

```diff
@@ -0,0 +1,30 @@
+| bgcAlt            | boolean                                                    | false         | false    | if true, appends `baseClass` with `--bgc-alt`                                                                                                                                                                                                                                                                        |
+| clickAct          | Function                                                   | null          | false    | Callback function that is trigger clicking on a row                                                                                                                                                                                                                                                                  |
+| filterOpts        | Array&lt;{<br>  text: string,<br>  value: string,<br>}&gt; | []            | false    | Array of object that are mapped to create filter options in the `<FilterComponent />`.<br>Value is placed in `predefined` prop of `<FilterComponent />`<br>When clicking on the `Filter Component` they are appear for selection<br>Clicking on a `Filter Option` will append the value to the search bar            |
+| tableTitle        | string                                                     | ''            | false    | String of `SimpleList` table                                                                                                                                                                                                                                                                                         |
+| sections          | boolean                                                    | false         | false    | If true `<Sections />` component will be rendered and if false `<Body />`<br> component will be rendered                                                                                                                                                                                                             |
+| sectionTarget     | string                                                     | ''            | false    | TODO: investigate the purpose of `sectionTarget`                                                                                                                                                                                                                                                                     |
+| sectionTitleKeys  | string&#91;&#93;                                           | []            | false    | TODO: investigate the purpose of `sectionTitleKeys`                                                                                                                                                                                                                                                                  |
```

Comment:
The title of the inner section of the list, only used in combination with `sections`